### PR TITLE
Fix Hyper-V TIME cases for RHEL 8 and AMD host

### PR DESCRIPTION
**Pull request by lixuemin2016**
Three minor updates:
1. TIME-CLOCKEVENT
   when unblind the clockevent for AMD host, guest console will hang. Refer to bug https://bugzilla.redhat.com/show_bug.cgi?id=1762681, this is low priority bug since customer may not unbind it, but it will block lisav2 test.
2. TIME-CLOCKSOURCE
For intel cpu, it will show constant_tsc when grep /proc/cpuinfo, for AMD host, it will show tsc. In order to combine together, only check tsc here.
3. TIMESYNC-NTP
[[ $os_release = 8.* ]] is false when $os_release=8, also for rhel 8, skip NTP testing.

Test Result:
**Before Fix**
10/14/2019 02:31:10 : [INFO ] TIMESYNC-NTP ended running with status: ABORTED.
10/14/2019 02:31:10 : [INFO ] TIME-CLOCKSOURCE started running.
10/14/2019 02:35:45 : [INFO ] TIME-CLOCKSOURCE ended running with status: FAIL.
10/14/2019 02:35:45 : [INFO ] TIME-CLOCKEVENT-UP started running.  --> **Cannot connect with VM via ssh**

**After Fix**
```
  ID TestArea             TestCaseName                                                                TestResult TestDu
ation(in minutes)
-----------------------------------------------------------------------------------------------------------------------
------------------
   1 CORE                 TIMESYNC-NTP                                                                   SKIPPED
         1.06
   2 CORE                 TIME-CLOCKSOURCE                                                                  PASS
         1.84
   3 CORE                 TIME-CLOCKEVENT-UP                                                                PASS
```

### CODE REVIEW

**Review by LiliDeng — APPROVED** [review]

**Comment by juhlee-microsoft** [comment]
Let me check out with other cases and get back to here. Please wait for my sign-off.

File: Testscripts/Linux/TIME-CLOCKSOURCE.sh
Comment (juhlee-microsoft) [inline Testscripts/Linux/TIME-CLOCKSOURCE.sh]: LogErr

**Review by juhlee-microsoft — CHANGES_REQUESTED** [review]

**Comment by juhlee-microsoft** [comment]
> Let me check out with other cases and get back to here. Please wait for my sign-off.

Look good for change tests with other distro. Please fix one issue, and then I will sign off. @lixuemin2016

**Review by juhlee-microsoft — APPROVED** [review]

**Pull request merged**
